nwb_explorer/nwb_model_interpreter/nwb_model_interpreter.py

```diff
@@ -70,7 +75,20 @@ def assign_name_to_types(self, pynwb_obj):
     def add_variables_to_type(self, pynwb_obj, geppetto_composite_type, geppetto_common_library_access):
         ''' Use this function to add variables to a geppetto compositeType '''
         
-        if isinstance(pynwb_obj, TimeSeries):
+        if isinstance(pynwb_obj, ImageSeries):
+            # TODO lazy fetching with importValue
+            
+            for key, value in pynwb_obj.fields.items():
+                if isinstance(value, (str, int, float)):
```

Comment:
It looks like we can improve the reuse the code on the generation of metadata. Not on this pr anyway, we can maybe plan a refactor.

Reply:
Yes, I extracted the code as a function.